sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -804,6 +827,10 @@ public enum Method {
 
       abstract Builder<T> setParseFn(SerializableFunction<SchemaAndRecord, T> parseFn);
 
+      abstract Builder<T> setReaderDatumFactory(AvroSource.DatumReaderFactory<T> factory);
```

Comment:
this should be named `setDatumReaderFactory` no?

Reply:
oh yeah